src/io/flutter/sdk/FlutterSdkManager.java

```diff
@@ -5,10 +5,12 @@
  */
 package io.flutter.sdk;
 
+import com.google.common.base.Objects;
```

Comment:
I haven't checked but is this adding a new dependency?  (I guess if travis is happy, we're OK.)